### DISCUSSION TITLE
Auto-Login not working on many sites after latest update?

### DISCUSSION THREAD

**Discussion by Lunchtime0614** [dani-garcia/vaultwarden#4052] [General]
I have Vaultwarden running in a Docker container and use cloudflare tunnel to access it remotely.  Ever since the latest release my browser extension(s) have been reporting that they can't enter the login credentials and I have to manually input them.  Is this a known issue (I didn't see anyone reporting this already) or do I need to do something (I saw the note about websockets/reverse proxy but I'm not sure that is even related to this issue).

Oddly it does work on some sites, but not on others.

**Comment by BlackDex** [comment]
That is strange, i have not yet seen any issue with this.
But since this is a client thingy, it's not something the Vaultwarden backend can fix or control.

**Reply by Lunchtime0614** [reply]
It's very strange, i've tried it on multiple machines, multiple browsers, OS's and the experience is the same - some auto-login some don't.

**Reply by Lunchtime0614** [reply]
Issue is that the extensions updated and broke auto-login for many sites...downgrading to the previous extension version solved the issue.

**Reply by BlackDex** [reply]
Since this is a client issue, it's not something Vaultwarden can fix, i would suggest to check the bitwarden/client repo for know issue.

**Reply by Lunchtime0614** [reply]
Seems to be some sort of bug in bitwarden for selfhosted users and/or bitwarden.eu - - https://github.com/bitwarden/clients/issues/6832#issuecomment-1802707780

**Reply by BlackDex** [reply]
Should be fixed once #4056 is merged

**Reply by Lunchtime0614** [reply]
@BlackDex  - great....I'm not knowledgeable on how github works etc...but if/when the official bitwarden repo is fixed does that also get fixed here?  Meaning does v2 get re-enabled - I'm not sure exactly of what v2 does compared to v1 but it sounds like there's new and/or improved functionality.

**Reply by BlackDex** [reply]
We will keep an eye on it. But for now leave the v2 disabled.

If you want, you can test it right now by using the `testing` tagged Vaultwarden image. That has the fix in it.

Would be great if you or others can confirm.

**Reply by ZorkATD** [reply]
Hi,
I tried the testing version with a self-hosted vault, it seems that the autofill problem is corrected.
Thanks.

**Reply by BlackDex** [reply]
Thx for checking!

**Reply by stefan0xC** [reply]
@BlackDex Could we do a hotfix release? Also so that the `alpine` tag is updated too.

**Reply by BlackDex** [reply]
I want to fix the export/deauth when using `Login with device` too before that. But yes, that was the idea.

**Reply by BJReplay** [reply]
Thanks, @htims05 for reporting this, and @BlackDex for fixing it.

I'd noticed this - only on one site - but I'm in-between password managers - I'm currently still using LastPass families subscription while I get my vaultwarden set up robust enough to migrate my elderly parents to Bitwarden, so I just used LastPass for this site.

Now I don't have to!
